htdocs/index.php

```diff
@@ -30,6 +30,15 @@
         $tpl->assign('message', $translate->t('You can find everything you need to go Geocaching ...', '', '', 0));
     }
 
+    // get HQ Message from DB
+    $aGetHqMessage = $connection->fetchAssoc(
```

Comment:
Can't we create a entity, repository and service for this und call it here?

Reply:
At first I kept to the outdated code format in order not to introduce a new format in the old code.